torch/custom_class.h

```diff
@@ -55,8 +55,8 @@ class class_ {
   /// see this class exposed as in Python and TorchScript. For example, if
   /// you pass in "MyStack" here, the class will appear as
   /// `torch.classes.MyStack` in both Python and TorchScript.
-  explicit class_(const std::string& className) : className(std::move(className)) {
-    qualClassName = topModule + "." + parentModule + "." + className;
+  explicit class_(const std::string& namespaceName, const std::string& className) {
```

Comment:
this needs to sanitize namespaceName/className or users can get really weird errors.

Reply:
random idea: shall we support "namespace::foo" syntax similarly to op definition API?

and yea, error out if name contains '.' or '::' is good to prevent mistakes

Reply:
@dzhulgakov that's an interesting point: I would so for now no because we have two disjoint naming conventions in the system:

* the op namespace thing (e.g. 'aten::foo'`
* QualifiedName (foo.bar.baz.Quux)

It may be something that we want to bring up with the op registration effort. Can we reconcile these things?